crypto/err/err.c

```diff
@@ -839,17 +721,7 @@ const char *ERR_reason_error_string(unsigned long e)
 
 void ERR_remove_thread_state(const CRYPTO_THREADID *id)
```

Comment:
So, I left the function but changed the argument to `void *unused` since `CRYPTO_THREADID` is deprecated.